jdbcSource/src/test/java/io/vantiq/extsrc/jdbcSource/TestJDBC.java

```diff
@@ -649,9 +665,74 @@ public void testMaxMessageSize() throws VantiqSQLException {
         // Delete the Source from VANTIQ
         deleteSource();
     }
+
+    @Test
+    public void testAsynchronousProcessing() throws InterruptedException {
+        // Only run test with intended vantiq availability
+        assumeTrue(testAuthToken != null && testVantiqServer != null);
+        assumeTrue(testDBUsername != null && testDBPassword != null && testDBURL != null && jdbcDriverLoc != null);
+
+        // Check that Source, Type, Topic, Procedure and Rule do not already exist in namespace, and skip test if they do
+        assumeFalse(checkSourceExists());
+        assumeFalse(checkTypeExists());
+        assumeFalse(checkTopicExists());
+        assumeFalse(checkProcedureExists());
+        assumeFalse(checkRuleExists());
+
+        // Setup a VANTIQ JDBC Source, and start running the core
+        setupSource(createSourceDef(true));
```

Comment:
should we also have a test when the optional parameters (max...Threads or whatever they're called) aren't provided?